composer.json

```diff
@@ -34,8 +33,8 @@
         "behat/behat":           "^3.0.11",
         "bossa/phpspec2-expect": "~1.0",
         "symfony/filesystem":    "~2.1",
-        "symfony/process":       "~2.1",
-        "phpunit/phpunit":       "~4.4"
+        "phpunit/phpunit":       "~4.4",
+        "symfony/process":       "2.6"
```

Comment:
Ah, i meant to lock it at `2.6.*`. `2.7` requires `php-5.3.9` but travis still runs tests against `php-5.3.3` (so composer install would fail using `2.7` on `5.3.3`). I've updated the PR.